### DISCUSSION TITLE
Welcome to OpenFarmCarbonTracker Discussions!

### DISCUSSION THREAD

**Discussion by kristofferronn** [euraf/OpenFarmCarbonTracker#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EURAF-Policy** [comment]
Thanks for creating this site Kristoffer.  Will you be able to go to the make a presenation on the OpenFarmCarbonTracker at the European [Carbon Farming Summit ](https://www.carbonfarmingsummit.eu/) in Dublin on 3-5 March 2025?  Can we have a Google Meet before then so you can record a demonstration?
